Yosemite/Yosemite/Model/Mocks/ActionHandlers/MockCouponActionHandler.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Storage
+import Networking
+
+struct MockCouponActionHandler: MockActionHandler {
+
+    typealias ActionType = CouponAction
+
+    let objectGraph: MockObjectGraph
+    let storageManager: StorageManagerType
+    let store: CouponStore
```

Comment:
nit: I think we can remove this `store` as we don't use it.

Reply:
Thanks! https://github.com/woocommerce/woocommerce-ios/pull/14477/commits/5cffd2c3f4260fbc8e1123204c77435a330e4756